scripts/model_merger.py

```diff
@@ -85,11 +86,16 @@ def convert_fsdp_checkpoints_to_hfmodels():
                             map_location='cpu')
     pivot_key = sorted(list(state_dict.keys()))[0]
     weight = state_dict[pivot_key]
-    assert isinstance(weight, torch.distributed._tensor.DTensor)
-    # get sharding info
-    device_mesh = weight.device_mesh
-    mesh = device_mesh.mesh
-    mesh_dim_names = device_mesh.mesh_dim_names
+
+    if isinstance(weight, torch.distributed._tensor.DTensor):
```

Comment:
In Pytorch 2.6, DTensor is moved to torch.distributed.tensor.DTensor. Would you mind also fix this as well? Thanks!